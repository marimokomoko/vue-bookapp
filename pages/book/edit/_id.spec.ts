import { mount } from '@vue/test-utils'
import Edit from '@/pages/book/edit/_id.vue'
describe('画面レイアウト', ()=>{
    it('スナップショットテスト', ()=>{
        const wrapper = mount(Edit)
        expect(wrapper.html()).toMatchSnapshot()
    })
})