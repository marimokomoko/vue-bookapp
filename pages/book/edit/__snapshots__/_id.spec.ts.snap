// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`画面レイアウト スナップショットテスト 1`] = `
"<div>
  <h1>BookEdit</h1>
  <v-row class=\\"mt-4\\">
    <v-col cols=\\"12\\">
      <v-card class=\\"mx-auto\\">
        <v-row>
          <v-col cols=\\"4\\">
            <v-img class=\\"ml-3\\"></v-img>
          </v-col>
          <v-col cols=\\"8\\">
            <v-card-title></v-card-title>
            読んだ日:
            <v-menu nudge-right=\\"40\\" transition=\\"scale-transition\\" offset-y=\\"\\" min-width=\\"auto\\">
              <v-date-picker locale=\\"jp-ja\\" day-format=\\"function (date) {
        /* istanbul ignore next */
        cov_uxddy44o().f[12]++;
        cov_uxddy44o().s[48]++;
        return new Date(date).getDate();
      }\\"></v-date-picker>
            </v-menu>
            メモ:
            <v-textarea class=\\"mx-2\\">
            </v-textarea>
            <v-card-actions>
              <v-btn color=\\"secondary\\" to=\\"/book\\">一覧に戻る</v-btn>
              <v-btn color=\\"info\\">保存する</v-btn>
            </v-card-actions>
          </v-col>
        </v-row>
      </v-card>
    </v-col>
  </v-row>
</div>"
`;
